packages/fxa-settings/src/components/App/index.test.tsx

```diff
@@ -330,5 +340,10 @@ describe('App component', () => {
       );
       expect(history.location.pathname).toBe('/settings');
     });
+
+    it('redirects ChangePassword', async () => {
+      await history.navigate(HomePath + '/change_password');
+      expect(history.location.pathname).toBe('/settings/create_password');
+    });
```

Comment:
Right, they can't use that page at all since it requires a current password. This is to safeguard if a user tries to hit the URL directly to send them to the proper form (create password), like if they've clicked on a link to the change password page from a SUMO article.